### PR TITLE
[TS] LPS-132641

### PR DESCRIPTION
**Pull request by ChrisKian**
https://issues.liferay.com/browse/LPS-132641

Please let me know if you have any questions, thanks!

### CODE REVIEW

**Review by drewbrokke — COMMENTED** [review]
@ChrisKian apart from my single comment, the code here looks good.

@dejuknow please [LPS-132641](https://issues.liferay.com/browse/LPS-132641) for the scenario. Basically, we want to be able to opt-out of requiring the password to change username and email address when SSO is enabled. It seems reasonable to me.

File: modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/user_display_data.jsp [hunk 1]
```diff
@@ -180,7 +180,7 @@ User selUser = (User)request.getAttribute(UsersAdminWebKeys.SELECTED_USER);
 	<portlet:param name="mvcPath" value="/user/password_verification.jsp" />
 </portlet:renderURL>
 
-<c:if test="<%= selUser != null %>">
+<c:if test="<%= (selUser != null) && PrefsPropsUtil.getBoolean(company.getCompanyId(), PropsKeys.COMPANY_SECURITY_UPDATE_PASSWORD_REQUIRED) %>">
```
[the remark this answers is not in the document]
Comment (ChrisKian) [reply modules/apps/users-admin/users-admin-web/src/main/resources/META-INF/resources/user/user_display_data.jsp]: Yes @drewbrokke, you're right.  I've force-pushed the changes

**Comment by dejuknow** [comment]
@ChrisKian, @drewbrokke: Makes sense to me. Let's move forward.

**Comment by drewbrokke** [comment]
ci:forward

**Pull request closed without merging**